auv_sim/auv_sim_description/taluy_sim_description/urdf/sensors.urdf.xacro

```diff
@@ -13,4 +13,6 @@
     topic_name="cameras/cam_bottom" />
   <xacro:xsens_mti_g710_sim name="${namespace}/base_link/imu" namespace="${namespace}"
     static_reference_frame="${inertial_reference_frame}" />
-</robot>
+  <xacro:altimeter_sim name="${namespace}/base_link/altimeter" parent="${namespace}/base_link"
```

Comment:
Instead of attaching this to a different frame, you can use the dvl frame. Our dvl frame should be  oriented Z down if I'm Not mistaken. If that replicates your usage here maybe use that. If not, just set your parent to the dvl link and orient however you want and set it "altimeter" frame like you do here. No need to transform all the way from the origin. I'm not sure how our altitude message is sent. If it is sent with a header, then its frame id also should be matched, forcing you to mount this sensor exactly on whatever frame that is. Feel free to investigate this

Reply:
This makes sense, now we use dvl frame. If my observations are right, the dvl frame is not oriented Z down, so I added the necessary orientation for the altimeter.

Altitude data is sent as float32, there is no header implementation. There is some hard coded offset in the node its used, maybe making an actual transformation using the dvl frame can be a future improvement in that node.

Reply:
Right good catch. It seems publishing the altitude from the dvl frame is a bit useless since the altitude information (if we wanted to use it) would be better with base to bottom distance. So I  don't know why I used the dvl altitude without the header frame but whatever, yours look good and reflect the current codebase. If we decide to improve the codebase in the future, we update this node too.